sdk/communication/azure-communication-common/src/main/java/com/azure/android/communication/common/CommunicationUserIdentifier.java

```diff
@@ -8,17 +8,17 @@
 /**
  * Communication identifier for Communication Services Users
  */
-public class CommunicationUser extends CommunicationIdentifier {
+public class CommunicationUserIdentifier extends CommunicationIdentifier {
```

Comment:
Is this changing in Java too? There it would need changes to Chat as well, since the Builder.credential takes CommunicationUser

Reply:
Yes. The same change is happening in Java as well. c.f.
https://github.com/Azure/azure-sdk-for-java/pull/18064